web/hooks.go

```diff
@@ -171,12 +172,13 @@ func runTests(logger *zap.Logger, db database.Database, runner ci.Runner, repo *
 		logger.Error("Failed to extract results from log", zap.Error(err))
 		return
 	}
+	log.Println("hooks: RunTests got result from CI with score: ", result.TotalScore())
```

Comment:
If these log statements are meant to be useful in production, we can use `logger.Debug()` as used elsewhere in this function.

Reply:
No, they are absolutely not meant for production. I will remove them when I am ready with understanding what is wrong with submission score parsing. Hopefully tomorrow.